app/services/albums.js

```diff
@@ -17,6 +17,18 @@ exports.getAlbums = qs => {
   });
 };
 
+exports.getAlbums = () => {
+  const options = {
+    method: 'GET',
+    uri: `${urlAlbumApi}${albumsEndpoint}`,
+    json: true
+  };
+  return request(options).catch(() => {
+    logger.error('Error trying to consume album API');
```

Comment:
When logging a handled error, it's usually a good idea to log at least a part of said error, to be able to understand ir just by reading the logs. Here you should do something like 
``.catch(error =>  { logger.error(`Looks like something went wrong. Details: ${error}`) });``